pkg/storage/leveldbstore/recovery_test.go

```diff
@@ -10,11 +10,12 @@ import (
 	"fmt"
 	"testing"
 
+	"slices"
```

Comment:
This should be in the upper group. The same below.

Reply:
I ran the `gofumpt` formatter and they got re-arranged now